service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/disk/ControlledAzureDiskResource.java

```diff
@@ -71,6 +73,14 @@ public ControlledAzureDiskResource(DbResource dbResource) {
     validate();
```

Comment:
Aside: I wonder if we'd benefit from an ugly separation of the parts of the controlled resource names, e.g. `Controlled_Azure_DiskResource`. It'd be easier to parse at a glance, as we'd know which parts went to which concept.